src/gui_common/CustomRichTextLabel.cs

```diff
@@ -124,7 +134,12 @@ private void ParseCustomTags()
             return;
         }
 
+        var old = rawTranslatedMarkup;
         var translated = TranslationServer.Translate(extendedBbcode);
+        if (translated == old)
```

Comment:
I think this breaks custom tags in any rich text labels that directly set the text on them and doesn't use translations.